README.md

```diff
@@ -191,9 +191,9 @@ String that applies `rtl` or `ltr` form to the stack.
 
 When set to `false` the back swipe gesture will be disabled when the parent `Screen` is on top of the stack. The default value is `true`.
 
-#### `translucent` (iOS only)
+#### `translucent`
 
-When set to `true`, it makes native navigation bar on iOS semi-transparent with blur effect. It is a common way of presenting a navigation bar introduced in iOS 11. The default value is `false`.
+When set to `true`, it makes native navigation bar on iOS semi-transparent with blur effect, while on Android is fully transparent. It is a common way of presenting a navigation bar introduced in iOS 11. The default value is `false`.
```

Comment:
I also think this is not true for either of the platforms. All it does is make it possible for the content to be visible under the header due to not rendering content below the header but under it. It does not change the transparency by itself since it is controlled by `backgroundColor`. Or am I missing something?

Reply:
Yes, you are right. Updated.